src/cute_graphics.cpp

```diff
@@ -1312,12 +1312,32 @@ void cf_mesh_set_attributes(CF_Mesh mesh_handle, const CF_VertexAttribute* attri
 
 void cf_mesh_update_vertex_data(CF_Mesh mesh_handle, void* data, int count)
 {
-	// @TODO Destroy/recreate for a larger size.
-
 	// Copy vertices over to the driver.
 	CF_MeshInternal* mesh = (CF_MeshInternal*)mesh_handle.id;
 	CF_ASSERT(mesh->attribute_count);
 	int size = count * mesh->vertices.stride;
+
+	// Resize buffers if necessary
+	if (size > mesh->vertices.size)
+	{
```

Comment:
Nit: Brace on same line as predicate

Reply:
Speaking of this, @RandyGaul how do you feel about adding linters and formatters so the code is always consistent?

Reply:
Ehh not the biggest fan since it takes a lot of time to setup and maintain

Reply:
But then it makes life easier, and with github actions it can be validates with each PR.